basicswap/templates/wallet.html

```diff
@@ -1,515 +1,792 @@
-</section>
-
-<section>
-  <div class="pl-6 pr-6 pt-0 pb-0 h-full overflow-hidden">
-    <div class="pb-6 border-coolGray-100">
-      <div class="flex flex-wrap items-center justify-between -m-2">
-        <div class="w-full pt-2">
-          <div class="container mt-5 mx-auto">
-            <div class="pt-6 pb-8 bg-coolGray-100 dark:bg-gray-500 rounded-xl">
-              <div class="px-6">
-                <div class="w-full mt-6 pb-6 overflow-x-auto">
-                  <table class="w-full min-w-max text-sm">
-                    <thead class="uppercase">
-                      <tr class="text-left">
-                        <th class="p-0">
-                          <div class="py-3 px-6 rounded-tl-xl bg-coolGray-200 dark:bg-gray-600">
-                            <span class="text-xs text-gray-600 dark:text-gray-300 font-semibold">Options</span>
+  </section>
+  <section>
+    <div class="pl-6 pr-6 pt-0 pb-0 h-full overflow-hidden">
+      <div class="border-coolGray-100">
+        <div class="flex flex-wrap items-center justify-between -m-2">
+          <div class="w-full pt-2">
+            <div class="container mt-5 mx-auto">
+              <div class="pt-6 pb-6 bg-coolGray-100 dark:bg-gray-500 rounded-xl">
+                <div class="px-6">
+                  <div class="w-full mt-6 pb-6 overflow-x-auto">
+                    <table class="w-full min-w-max text-sm">
+                      <thead class="uppercase">
+                        <tr class="text-left">
+                          <th class="p-0">
+                            <div class="py-3 px-6 rounded-tl-xl bg-coolGray-200 dark:bg-gray-600"> <span class="text-xs text-gray-600 dark:text-gray-300 font-semibold">Options</span> </div>
+                          </th>
+                          <th class="p-0">
+                            <div class="py-3 px-6 rounded-tr-xl bg-coolGray-200 dark:bg-gray-600"> <span class="text-xs text-gray-600 dark:text-gray-300 font-semibold">Input</span> </div>
+                          </th>
+                        </tr>
+                      </thead>
+                      <tr class="opacity-100 text-gray-500 dark:text-gray-100">
+                        <td class="py-4 pl-6 bold"> <span class="inline-flex align-middle items-center justify-center w-9 h-10 bg-white-50 rounded"> <img class="h-7" src="/static/images/coins/{{ w.name }}.png" alt="{{ w.name }}"> </span> {{ w.name }} Balance: </td>
+                        <td class="py-3 px-6" data-coinname="{{ w.name }}">{{ w.balance }} {{ w.ticker }} </td>
+                        {% if w.cid == '3' %}
+                        {# LTC #}
+                      <tr class="opacity-100 text-gray-500 dark:text-gray-100">
+                        <td class="py-4 pl-6 bold w-1/4"> <span class="inline-flex align-middle items-center justify-center w-9 h-10 bg-white-50 rounded"> <img class="h-7" src="/static/images/coins/{{ w.name }}.png" alt="{{ w.name }}"> </span> {{ w.name }} MWEB Balance: </td>
```

Comment:
Not sure where the correct line is, but

Issue LTC:
1. "balance" = full wallet balance (transparent + mweb). 
2. "mweb balance" = way off.

expected: 
1. "Balance" should be transparent only
2. "Mweb balance" = "full wallet balance" minus "transparent" balance

Reply:
Checking, thanks.